api/v1/standalone-dns-proxy/standalone-dns-proxy.proto

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+syntax = "proto3";
+
+package standalonednsproxy;
+
+option go_package = "github.com/cilium/cilium/api/v1/standalonednsproxy";
+
+service FQDNData {
+  rpc SubscribeToDNSPolicies(stream  DNSPoliciesResult) returns (stream DNSPolicies) {}
+  rpc UpdatesMappings(FQDNMapping) returns (UpdatesMappingsResult) {}
+}
+
+message DNSPoliciesResult {
+    bool success = 1;
+    string request_id = 2;
+}
+
+message FQDNMapping {
+    string FQDN = 1;
+    repeated bytes IPS = 2;
+    uint32 TTL = 3;
```

Comment:
If there's delay in the processing (on either side), this TTL seems ~meaningless. Should the SDS convert this into a timestamp?

Reply:
this TTL is received from dnsserver and cilium agent requires it to garbage collect expired IPs. SDP doesn't use dns TTL.

Reply:
I understand where the TTL comes from, but the TTL is a number of seconds, which only retains meaning when received within reasonable time. Imagine the agent being down, if we replay this after, say, 5 minutes, the TTL is way off.

Reply:
good point. UpdateMapping would not retry on connection error. wanted to keep behavior same as agent dnsproxy where it updates mapping to agent with TTL and doesn't account for agent processing time or time incurred between dnsreponse and updatemapping call.